ts/tests/unit_tests/metrics_yaml_testing/metric_cache_unit_test.py

```diff
@@ -0,0 +1,1270 @@
+"""
+Unit testing for ts/metrics/metric_cache_abstract.py,
```

Comment:
Just confirming the intent is to run these tests in our regression tests?

How long do all the tests take to run now? If it's short like 1-3 min I think we should add them to our CI